### ISSUE TITLE
gitea show wrong version under FreeBSD

### ISSUE THREAD

**Issue by Oxyaxion** [go-gitea/gitea#10774]
After an update from pkg repository to 1.11.2

```
root@gitea:/usr/local/etc # pkg info | grep gitea
gitea-1.11.2                  Compact self-hosted Git service
```

```
root@gitea:/usr/local/etc # /usr/local/sbin/gitea --v
Gitea version 1.9.0-dev built with go1.13.5
```

Same thing in web interface (wrong version).

**Comment by mrsdizzie** [comment]
1.9.0-dev is just the default of what is in go.main if the version isn't included in LDFLAGS:

https://github.com/go-gitea/gitea/blob/e2da9cd21f0d87696b9b72d8fb704f897e1920e7/main.go#L32

(its been changed to just 'development' for the next release and going forward).

I'm not really sure how they build it but it should be set like this example from the Gitea MAKEFILE if they are modifying LDFLAGS:

https://github.com/go-gitea/gitea/blob/e5a008f7b2298977405fdbd806d3fa5ff4fd837e/Makefile#L63

**Comment by lunny** [comment]
This should be a package problem.

**Comment by stblassitude** [comment]
Already fixed in package version 1.11.3, see [bug 244898](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=244898).

Please feel free to report issues wit the FreeBSD package via the FreeBSD bugzilla or directly to me.
